src/App.vue

```diff
@@ -158,10 +160,16 @@ export default {
     },
     async getFeed(channelID) {
       this.loading = true;
-      const feed = await loadFeed(channelID);
-      this.loading = false;
-      this.addChannel(feed);
-      this.addVideos(feed);
+      try {
+        const feed = await loadFeed(channelID);
+        this.addChannel(feed);
+        this.addVideos(feed);
+      } catch (e) {
+        this.errorMessage = e.message;
+        setInterval(() => { this.errorMessage = ''; }, 10 * 1000); // Automatic dismiss after 10 seconds
```

Comment:
shouldn't this be `setTimeout`? `setInterval` resets it **every 10 seconds**.

Easily replicatable, cause an error, wait for the alert to be dismissed and then wait a few seconds before causing another feed pulling error. The second alert will only stay for a handful of seconds, even less maybe.

Reply:
Jup, should be `setTimeout`, my bad. If we're using snackbars, we should also find a way to show several errors at once and let them timeout by themselves.

Reply:
maybe doing `errorMessages` as an Array, pushing into it to add new errors? If the old ones already were dismissed by `timeout` only new ones should be created anyway.

This way, at some point later in time, we could also show an error log in the frontend, of sorts.